core/controllers/classroom_test.py

```diff
@@ -84,24 +83,6 @@ def test_get(self) -> None:
         topic_services.save_new_topic(admin_id, public_topic)
         topic_services.publish_topic(topic_id_2, admin_id)
 
-        csrf_token = self.get_new_csrf_token()
-        new_config_value = [{
-            'name': 'math',
-            'topic_ids': [topic_id_1, topic_id_2, topic_id_3],
-            'course_details': 'Course details for classroom.',
-            'topic_list_intro': 'Topics covered for classroom',
-            'url_fragment': 'math',
-        }]
-
-        payload = {
-            'action': 'save_config_properties',
-            'new_config_property_values': {
-                config_domain.CLASSROOM_PAGES_DATA.name: (
-                    new_config_value),
-            }
-        }
-        self.post_json('/adminhandler', payload, csrf_token=csrf_token)
```

Comment:
Quick question, If we are not updating the data of classroom pages via the config controllers then how are we doing that? If we are removing the updation of the classroom-data from the test here, should we not also replace it with the method we are currently using?
Thanks

Reply:
Ah! I see, I somehow missed that. Thanks for the response.